charts/member-agent/templates/deployment.yaml

```diff
@@ -84,6 +87,11 @@ spec:
           volumeMounts:
```

Comment:
change `{{- if not .Values.useCAAuth }}` to `{{- if or (not .Values.useCAAuth) (eq .Values.propertyProvider "azure") }}` on line 86